### DISCUSSION TITLE
Change Font Color and Gauge Color

### DISCUSSION THREAD

**Discussion by krevordelapaz** [Live-Charts/LiveCharts2#550] [Q&A] [answered]
Hi,

I'm trying to use the gauge control you made and want to change the color of the components highlighted on the picture. Is that even possible?

![image](https://user-images.githubusercontent.com/16677822/178753057-79faca5e-0ce8-4bb4-9536-04d7c5d762ba.png)

Thank you!

**Comment by beto-rodriguez** [comment, marked as answer]
This was missing in the docs! it is now included.

Try 

`.AddValue(30, "gauge value", SKColors.YellowGreen, SKColors.Red)`

**Comment by cmello74** [comment]
Is there a way to change the color of the gauge bar depending on it's value?      ex;  battery meter that is red when it drops below a certain voltage, but green when it's above it.
